hedera-node/hedera-addressbook-service-impl/src/main/java/com/hedera/node/app/service/addressbook/impl/schemas/V053AddressBookSchema.java

```diff
@@ -127,9 +127,9 @@ public void migrate(@NonNull final MigrationContext ctx) {
             if (nodeDetailMap != null) {
                 nodeDetail = nodeDetailMap.get(nodeInfo.nodeId());
                 if (nodeDetail != null) {
-                    nodeBuilder
-                            .serviceEndpoint(nodeDetail.serviceEndpoint())
-                            .grpcCertificateHash(nodeDetail.nodeCertHash());
+                    final Bytes hashBytes =
+                            Bytes.fromHex(nodeDetail.nodeCertHash().asUtf8String());
```

Comment:
This fix should be in `NodeMetadataHelper`. 0.53 Schema doesn't run in this release

Reply:
I completely missed that class.  Thank you for calling it out.